handlers/attachment/index.js

```diff
@@ -0,0 +1,56 @@
+var conf = require('../../conf/config');
+const express = require('express');
+const fs = require('fs');
+const path = require('path');
+const { listAttachments, fetchDoc, deleteAttachment } = require('./attachments');
+const multer = require('multer');
+const getUuid = require('uuid-by-string');
+const attachmentRouter = express.Router();
+
+// Let the upload middleware buffer the file in memory
+const storage = multer.memoryStorage();
+const upload = multer({ storage: storage });
+
+let configOptions = {};
+
+if (conf['attachment']) {
+    configOptions = Object.assign({}, conf['attachment']);
+}
+
+attachmentRouter.get('/:layer/:object/attachments/', listAttachments);
+attachmentRouter.get('/:layer/:object/attachments/:id', fetchDoc);
+attachmentRouter.post('/:layer/:object/deleteAttachments/', deleteAttachment);
+attachmentRouter.post('/:layer/:object/addAttachment', upload.single('attachment'), function (req, res, next) {
+    const dir = path.join(configOptions.filepath, req.params.layer, req.params.object, req.body.group);
```

Comment:
It looks like I don't get a `req.body` property at all when POSTing using `Content-Type: application/octet-stream`.